### PR TITLE
feat: Draft of insert instructions.

### PR DESCRIPTION
**Pull request by beausmith**
Distilling the content from [Jake and others](https://docs.google.com/document/d/1dey16-gAAZWriaRPeWVfD-QiZPsC-5RcH_e_-26uyLA/edit#) into a proposal. 

This is text-only for now. [View full pdf of 4 insert pages](https://github.com/votingworks/mail-ballot-manager/files/4935632/inserts-v5-2020-JUL-16.pdf)

image

### CODE REVIEW

File: client/src/screens/InsertsEditScreen.tsx [hunk 1]
```diff
@@ -19,23 +19,27 @@ The below box must be signed or your vote will not be counted. **You must sign i
...
 
-1. Eu mi bibendum neque egestas congue quisque egestas diam.
-2. Tempus quam pellentesque nec nam aliquam sem et.
-3. Egestas purus viverra accumsan in nisl nisi. Nulla malesuada pellentesque elit eget gravida cum.
-4. Lobortis mattis aliquam faucibus purus in massa tempor nec. Adipiscing enim eu turpis egestas pretium aenean pharetra magna.
-5. Dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Erat nam at lectus urna duis convallis.
+## 1. Mark Your Ballot
 
-2Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Nunc mattis enim ut tellus elementum. In iaculis nunc sed augue lacus viverra vitae. Sed augue lacus viverra vitae congue eu consequat ac. Accumsan lacus vel facilisis volutpat est. Purus sit amet volutpat consequat mauris. Ultricies leo integer malesuada nunc vel risus commodo viverra maecenas.
+Use a dark pen to completely fill in the oval to the left of your choices. If you make a mistake, contact your local election office for a new ballot.
```
Comment (benadida) [inline client/src/screens/InsertsEditScreen.tsx:26]: would it be clearer to just say "black or blue pen", in case someone thinks blue is not dark enough?

**Review by benadida — APPROVED** [review]
simple and nice. One comment.

File: client/src/screens/InsertsEditScreen.tsx (hunk 1 above)
Comment (beausmith) [reply client/src/screens/InsertsEditScreen.tsx:26]: The idea was not to be prescriptive as our scanning only requires darkening, not specifically a pen. Thus a pencil could be used successfully.

Comment (benadida) [reply client/src/screens/InsertsEditScreen.tsx:26]: My suggestion would be to be specific in what we recommend, and more lenient in what we accept. So while our scanner can handle pencil and what not, that's not good for durability, so we don't want to recommend that, but of course if a voter uses a pencil we don't want to fail hard. Also, "dark" can feel ambiguous to folks, and I think clarity helps. Black/blue pen are common. A precise recommendation will help people feel like they know they're following the rules.

Comment (beausmith) [reply client/src/screens/InsertsEditScreen.tsx:26]: Sounds solid.

**Pull request merged**